pandas/tests/extension/decimal_array/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .array import DecimalArray, DecimalDtype
+
+
+__all__ = ['DecimalArray', 'DecimalDtype']
```

Comment:
can you add here make_data ? (that are typically the ones that I use to test things out locally)